deployments/discovery-engine-chart/configmapfiles/discovery-engine/conf.yaml

```diff
@@ -0,0 +1,74 @@
+application:
+  name: knoxautopolicy
+  network:
+    operation-mode: 1                         # 1: cronjob | 2: one-time-job
+    cron-job-time-interval: "0h0m10s"         # format: XhYmZs
+    operation-trigger: 1000
+    network-log-from: "hubble"                # db|hubble|feed-consumer

```

Comment:
This configuration is wrong. Please check the latest config in src/conf/local-file.yaml